llama_stack/distribution/routers/routers.py

```diff
@@ -93,6 +98,10 @@ async def insert_chunks(
         chunks: List[Chunk],
         ttl_seconds: Optional[int] = None,
     ) -> None:
+        logcat.debug(
+            "core",
+            f"VectorIORouter.insert_chunks: {vector_db_id}, {len(chunks)} chunks, ttl_seconds={ttl_seconds}, chunk_ids={[chunk.id for chunk in chunks[:3]]}{' and more...' if len(chunks) > 3 else ''}",
```

Comment:
I don;t think there is an `id` field in the `Chunk` class.
It should be changed with `chunk.metadata['document_id']` instead

Reply:
Tracked https://github.com/meta-llama/llama-stack/issues/1355